tests/Functional/ExceptionTest.php

```diff
@@ -300,7 +301,7 @@ public function testConnectionExceptionSqLite() : void
         }
 
         // mode 0 is considered read-only on Windows
-        $mode = PHP_OS === 'Linux' ? 0444 : 0000;
+        $mode = in_array(PHP_OS, ['Linux', 'Darwin']) ? 0444 : 0000;
```

Comment:
What about inverting the logic to simplify?
```suggestion
        $mode = PHP_OS_FAMILY === 'Windows' ? 0444 : 0000;
```

And use a new constant introduced in PHP 7.2: https://php.net/manual/reserved.constants#PHP_OS_FAMILY